lib/ProductOpener/Images.pm

```diff
@@ -357,6 +359,17 @@ sub scan_code ($file) {
 	return $code;
 }
 
+sub get_selected_image_uploader ($product_ref, $imagefield) {
+
+	# Retrieve the product's image data
+	my $imgid = deep_get($product_ref, "images", $imagefield, "imgid");
+
+	# Retrieve the uploader of the image
```

Comment:
Could you add a check that $imgid is defined? Otherwise it could trigger warnings if we try to retrieve a hash value for an undef key I think.